src/simulation/dynamics/facetSRPDynamicEffector/facetSRPDynamicEffector.rst

```diff
@@ -104,24 +113,24 @@ the Sun as a gravitational body in the simulation to use this module.
     SRPEffector = facetSRPDynamicEffector.FacetSRPDynamicEffector()
     SRPEffector.ModelTag = "SRPEffector"
 
-#. The user is required to set the total number of spacecraft facets and the number of articulated facets. For example, if the user wants to create a spacecraft with 10 total facets, four of which articulate; the user would set these module variables to::
+#. The user is required to set the total number of spacecraft facets and the number of articulated facets. For example, if the user wants to create a spacecraft with 10 total facets, four of which articulate::
 
-    SRPEffector.numFacets = 10
-    SRPEffector.numArticulatedFacets = 4
+    SRPEffector.setNumFacets(10)
+    SRPEffector.setNumArticulatedFacets(4)

```

Comment:
Here we can add a ".. warning:" note to the documentation that setting the variables `numFacets` and `numArticuatedFacets` directly is depreciated and will be removed in Dec. 2025.  This gives users 1 year grace period to update their code.

Reply:
Done